runtime.md

```diff
@@ -37,17 +35,22 @@ The lifecycle describes the timeline of events that happen from when a container
 
 1. OCI compliant runtime's `create` command is invoked with a reference to the location of the bundle and a unique identifier.
    How these references are passed to the runtime is an implementation detail.
-2. The container's runtime environment (namespaces, mounts, etc.) MUST be created according to the configuration in [`config.json`](config.md).
+1. The container's runtime environment (namespaces, mounts, etc.) MUST be created according to the configuration in [`config.json`](config.md).
    If a new PID namespace is requested in the [`config.json`](config.md), a PID namespace MUST created at this time.
-   However, the user-specified code (from ['process'](config.md#process-configuration) MUST NOT be executed at this time.
+   However, the user-specified code (from [`process`](config.md#process-configuration) MUST NOT be run at this time.
    With the exception of the user-specified process, any updates to `config.json` after container is created MUST NOT affect the container.
-3. Runtime's `start` command is invoked with the unique identifier of the container.
-   The runtime MUST create and start the user-specified code, as specified by [`process`](config.md#process-configuration), in the container's PID namespace.
-   Any updates to the user-specified code in ['process'](config.md#process-configuration) after this point MUST NOT have any effect on the container.
-4. Additional actions such as pausing the container, resuming the container or signaling the container MAY be performed using the runtime interface.
-   The container MAY also error out, exit or crash.
-5. Irrespective of how the user defined process stops (i.e. PID 1 exits), once the PID namespace is deleted the container MUST be destroyed by undoing the steps performed during create phase (step 2).
-   For clarity, all namespaces that were created in step 2 MUST be deleted.
+1. Once the container is created additional actions MAY be performed based on the features the runtiem chooses to support.
```

Comment:
nit: “runtiem” → “runtime”, although see [my preferred reroll of this whole section][1].

[1]: https://github.com/opencontainers/runtime-spec/pull/384#issuecomment-214418730